packages/cozy-mespapiers-lib/docs/papersDefinitions.md

```diff
@@ -0,0 +1,77 @@
+The `papersDefinitions.json` file is the configuration file that controls part of the display and all the creation of a paper.
+
+Let's see how it is built:
+
+*Properties surrounded by `[]` are optional.*
+
+- `papersDefinitions`: {object[]} Contains all possible paper definitions.
+
+  - `label`: {string} Equal a label of [qualification](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/assets/qualifications.json).
+  - `icon`: {string} Name of the [icon](https://docs.cozy.io/cozy-ui/react/#!/Icon/11) to use in the suggestion display (placeholder).
+  - `[placeholderIndex]`: {number} Presence and position in suggestion (without filter).
+  - `[featureDate]`: {string} Date to be highlighted, when several date steps are present.
+  - *`[filenameModel]`: {string[]} Allows to structure the final name of the paper.* **(not completed)**
+    - *The accepted values are:*
+      - *`label`: Reference to the qualification label.*
+      - *`featureDate`: Reference to the highlighted date.*
+      - *`contactName`: Reference to the names of contacts referenced on the paper.*
+      - *`name`: Value of the attribute present in the [`information`](#information-field-attributes) step.*
+  - `maxDisplay`: {number} Specifies the number of this paper to be displayed once created.
+  - `connectorCriteria`: {object} Allows to propose the installation of a connector before the steps of creation of a paper.
+    - `[name]`: {string} Name of the connector.
+    - `[category]`: {string} Connector category.
+  - [`acquisitionSteps`](#steps-of-the-acquisitionsteps-property): {object[]} Contains the steps of the creation process.
+    - [`scan`](#step-scan) {object} Step to select a file (image/pdf).
+    - [`[information]`](#step-information) {object} Step to get more informations about this file.
+    - [`owner`](step-owner) {object} Step to select one or more contacts linked to this file.
+
+___
+
+## Steps of the `acquisitionSteps` property:
+
+- ### Step `scan`:
+  - `stepIndex`: {number} Position of the step.
+  - `model`: {string} Model used of the step (`scan`).
+  - `[multipage]`: {boolean} Allows you to repeat this step without limit.
```

Comment:
c'est à dire `without limit` ?

Reply:
Ce paramètre permet à l'utilisateur d'ajouter autant de fichiers qu'il le souhaites, via le bouton ci-dessous
On pourrait aussi dire `Allows to add as many files as the user wants` ?

![Capture d’écran 2022-05-09 à 14 23 22](https://user-images.githubusercontent.com/14182143/167409229-293d5c9d-7d00-49cb-ad1b-3620f5d7c0f3.png)

Reply:
Ah ! Ok pour ta proposition 👍